ddtrace/writer.py

```diff
@@ -134,6 +138,27 @@ def _target(self):
 
             time.sleep(1) # replace with a blocking pop.
 
+            # Logging the http errors
```

Comment:
api uses the httplib python library to send the data

The request comes from the lib with the 3 attributes status, reason, msg
https://github.com/python-git/python/blob/master/Lib/httplib.py#L943

It is then transmitted here
https://github.com/DataDog/dd-trace-py/blob/raphael/agent_http_request_errors/ddtrace/api.py#L79

And then here
https://github.com/DataDog/dd-trace-py/blob/raphael/agent_http_request_errors/ddtrace/api.py#L56

Actually no errors are raised in case of 400-599 http erros